docs/navigation-parameters-reference.md

```diff
@@ -645,9 +645,11 @@ navigation: {
 
 ### tabNav
 - **type**: boolean or Object
-- **description**: renders the children of the node as a horizontal navigation bar. Sub-children are not supported. When you categorize nodes you will get a drop-down menu in the horizontal navigation.
-- **since**: v0.7.0
-In the case the node has only one child, it's possible to configure if the horizontal navigation bar will be hidden automatically or not. To do so, the `tabNav` property must be an object with the property `{hideTabNavAutomatically:true|false}`.
+- **description**: renders the children of the node as a horizontal navigation bar. Sub-children are not supported. When you categorize nodes you will get a drop-down menu in the horizontal navigation. Set to `true` to show the horizontal navigation, or use the extra attributes to for more customization. 
+- **attributes**:
+  - **hideTabNavAutomatically**: boolean. In the case the node has only one child, it's possible to configure if the horizontal navigation bar will be hidden automatically or not. Set to true to hide the horizontal navigation bar and false otherwise. ( **since**: v0.7.0 )
+  - **showAsTabHeader**: boolean. if set on the node, it considers it as horizontal navigation header microfrontend. The node should be webcomponent based and it should have nested children to show on the horizontal navigation bar. (**since**: NEXTRELEASE )
```

Comment:
```suggestion
  - **showAsTabHeader**: boolean. If this attribute is set on the node, it will be considered as a horizontal navigation header micro frontend. The node should be [webcomponent-based](web-component.md) and it should have nested children to show on the horizontal navigation bar. (**since**: NEXTRELEASE )
```